src/proto/FunctionRpc.proto

```diff
@@ -10,6 +10,7 @@ option go_package ="github.com/Azure/azure-functions-go-worker/internal/rpc";
 package AzureFunctionsRpcMessages;
 
 import "google/protobuf/duration.proto";
+import "Dependency.proto";
```

Comment:
Lets put these in a new folder "definitions" as @kashimiz suggested.

Reply:
Following the conventions similar to https://github.com/googleapis/googleapis/tree/master/google/bytestream, move new definition to its own folder